changebot/blueprints/user_config.py

```diff
@@ -0,0 +1,36 @@
+"""Module to deal with user configuration for each repo."""
+
+# STDLIB
+import os
+
+# THIRD-PARTY
+import yaml
+
+
+# TODO: Figure out how to properly enforce default file because my
+#       Heroku-fu is weak.
+# TODO: Do we want to load this only once at the start of a bot session
+#       or do it on-demand within function?
+# TODO: When finalized, add config instructions to README.
+def get_bot_config(filename='astropybotrules.py'):
+    """
+    Load user configuration for bot.
+
+    Parameters
+    ----------
+    filename : str
+        Configuration file in YAML format. If not given, default is used.
+
+    Returns
+    -------
+    cfg : dict
+        Configuration parameters.
+
+    """
+    if not os.path.isfile(filename):
+        raise ValueError("I am afraid I can't do that.")
```

Comment:
Note to self: Was trying to be funny. But probably more useful to issue warning and return empty dictionary here.